assets/js/base/utils/address.ts

```diff
@@ -100,3 +102,53 @@ export const emptyHiddenAddressFields = <
 
 	return newAddress;
 };
+
+/*
+ * Formats a shipping address for display.
+ *
+ * @param {Object} address The address to format.
+ * @return {string | null} The formatted address or null if no address is provided.
+ */
+export const formatShippingAddress = (
+	address: ShippingAddress | BillingAddress
+): string | null => {
+	// we bail early if we don't have an address.
+	if ( Object.values( address ).length === 0 ) {
+		return null;
+	}
+	const shippingCountries = getSetting( 'shippingCountries', {} ) as Record<
+		string,
+		string
+	>;
```

Comment:
instead of using `as Record< string, string >` you can pass the generic type to `getSetting` like so:

```suggestion
	const shippingCountries = getSetting< Record< string, string > >(
		'shippingCountries',
		{}
	);
```

This can apply to the other instances of this too :)

Reply:
Thank you for the suggestion. I have updated the code.  👍